javaagent-spi/src/main/java/io/opentelemetry/javaagent/spi/exporter/SpanExporterFactory.java

```diff
@@ -21,7 +21,7 @@
    * @param config The configuration
    * @return An implementation of a {@link SpanExporter}
    */
-  SpanExporter fromConfig(Properties config);
+  SpanExporter fromConfig(Config config);
```

Comment:
let's not change this here, we are going to remove `SpanExporterFactory` and `MetricExporterFactory` in #2442, and changing this will create unnecessary churn for @jsuereth and others who are using it in the meanwhile

Reply:
🤷 ok.